main.py

```diff
@@ -1,105 +1,38 @@
-#!/usr/bin/env python3
-# Copyright 2015 Google Inc. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# [START app]
+import datetime
 import logging
-import psycopg2
+import os
+import json
 
 from flask import Flask
-from flask import jsonify
 from flask_cors import CORS
-
-# DB Connection
-conn = psycopg2.connect("host=34.97.189.196 dbname=photo user=root password=yori1234")
-cur = conn.cursor()
-
-  
-# test data
-dummy0 = {
-    "photo_group": "乃木坂46",
-    "photo_member": "樋口日奈",
-    "photo_costume": "2019 summer concert 1",
-    "photo_type": "ヨリ",
-    "photo_number": 1,
-    "photo_folder": "summer concert",
-    "photo_tag": []
-}
-
-dummy1 = {
-    "photo_group": "乃木坂46",
-    "photo_member": "樋口日奈",
-    "photo_costume": "2019 summer concert 1",
-    "photo_type": "チュウ",
-    "photo_number": 3,
-    "photo_folder": "summer concert",
-    "photo_tag": []
-}
-
-dummy2 = {
-    "photo_group": "乃木坂46",
-    "photo_member": "樋口日奈",
-    "photo_costume": "2019 summer concert 1",
-    "photo_type": "ヒキ",
-    "photo_number": 1,
-    "photo_folder": "summer concert",
-    "photo_tag": []
-}
-
-photos = [dummy0, dummy1, dummy2]
+import sqlalchemy
 
 app = Flask(__name__)
-CORS(app)
-app.config["JSON_AS_ASCII"] = False
```

Comment:
I guess we also need this because we have a lot of nonascii characters in data

Reply:
Oh, thanks for your advice.